src/app/container/info-tab/info-tab.component.html

```diff
@@ -267,14 +267,45 @@
           <!-- Public Topic -->
           <li *ngIf="isPublic">
             <span><strong matTooltip="Short description of the tool">Topic</strong>:&nbsp;</span>
-            <span>{{ tool?.topicSelection === TopicSelectionEnum.AUTOMATIC ? tool?.topicAutomatic : tool?.topicManual }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.AUTOMATIC">{{ tool?.topicAutomatic }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.MANUAL">{{ tool?.topicManual }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.AI">
+              {{ tool?.topicAI }}
+              <span
+                class="bubble ai-bubble"
+                matTooltip="This topic sentence was AI generated. For more information, please click the bubble to visit our docs"
```

Comment:
Hyperlinks are not supported in matTooltips so I had to make the whole bubble clickable. And the docs link will need to be edited once we make the docs for the AI topic sentences since it is currently linked to the main FAQ as a placeholder